Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/proxy/AuditRepositoryProxyWebServiceSecuredImpl.java

```diff
@@ -50,13 +50,14 @@
 
     private static final Logger LOG = Logger.getLogger(AuditRepositoryProxyWebServiceSecuredImpl.class);
 
-    private WebServiceProxyHelper oProxyHelper = new WebServiceProxyHelper();
+    private final WebServiceProxyHelper oProxyHelper = new WebServiceProxyHelper();
+    private AcknowledgementType result = new AcknowledgementType();
+    private LogEventSecureRequestType secureRequest = new LogEventSecureRequestType();
 
+    @Override
     public AcknowledgementType auditLog(LogEventRequestType request, AssertionType assertion) {
         LOG.debug("Entering AuditRepositoryProxyWebServiceSecured.auditLog(...)");
-        AcknowledgementType result = new AcknowledgementType();
 
-        LogEventSecureRequestType secureRequest = new LogEventSecureRequestType();
         if (request.getAuditMessage() == null) {
```

Comment:
Need to handle the scenario when request.getAuditMessage() == null what value should be populated for LogEventSecureRequestType .setAuditMessage().  
LogEventSecureRequestType is a jaxb class and has AuditMessage as a required field.